config/environments/development.rb

```diff
@@ -34,4 +34,16 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+      authentication: :plain,
+      address: 'smtp.mailgun.org',
+      port: 587,
+      domain: 'sandboxb5b0b0ca7f9a4977993479199f0c84ea.mailgun.org',
+      user_name: 'postmaster@sandboxb5b0b0ca7f9a4977993479199f0c84ea.mailgun.org',
+      password: 'b09efd309f24990fe1524b1e981839cd'
+  }
+
+  config.action_mailer.default_url_options = { host: '127.0.0.2:3000', port: 3000 }
```

Comment:
Line is too long. [83/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.